com.ibm.wala.core/src/test11Subjects/java/privateInterfaceMethods/PrivateInterface.java

```diff
@@ -0,0 +1,11 @@
+package privateInterfaceMethods;
+
+public interface PrivateInterface {//called RetT and GetT b/c it used to type T return/input
+  default <Object> void RetT(Object input) {
+    GetT(input);
+  }
+
+  private <Object> Object GetT(Object input) {
```

Comment:
```suggestion
  private Object GetT(Object input) {
```

Method is no longer generic